string Class
float64 probability
int64 xmin_Camera
int64 ymin_Camera
int64 xmax_Camera
int64 ymax_Camera

